squad/frontend/templates/squad/_builds_table.jinja2

```diff
@@ -53,4 +61,9 @@
 </a>
 {% endwith %}
 {% endfor %}
+<input type="button" ng-value="compare_button_label" class="btn btn-default" ng-click="compareBuilds('{{statuses.0.build.project.full_name}}')" />
 </div>
+
+{% block javascript %}
+<script type="module" src='{{static("squad/build.js")}}'></script>
+{% endblock %}
```

Comment:
see my comment above; I think we should not include here all of the code that only applies to a single build

Reply:
yeah, you're right. Created a new file called `build_list.js` that should contain all js for build list, then moved that controller in.